zboxcore/sdk/read_counter_ctr.go

```diff
@@ -2,40 +2,43 @@ package sdk
 
 import (
 	"sync"
-
-	"github.com/0chain/gosdk/zboxcore/blockchain"
 )
 
 type blobberReadCounter struct {
 	m  map[string]int64
 	mu *sync.RWMutex
 }
 
-var brc blobberReadCounter
+var brc *blobberReadCounter
 
 func InitReadCounter() {
-	brc = blobberReadCounter{
-		m:  make(map[string]int64),
-		mu: &sync.RWMutex{},
+	if brc == nil {
+		brc = &blobberReadCounter{
+			m:  make(map[string]int64),
+			mu: &sync.RWMutex{},
+		}
 	}
 
 }
 
-func setBlobberReadCtr(blobber *blockchain.StorageNode, ctr int64) {
+func setBlobberReadCtr(clientID, blobberID string, ctr int64) {
```

Comment:
can this be allocationID + blobberID instead? 
gosdk can be initialized with one wallet so it would always be same client. But gosdk can work on many allocations.

Reply:
It actually is a good point.